doc/rtd/topics/bugs.rst

```diff
@@ -0,0 +1,95 @@
+.. _reporting_bugs:
+
+Reporting Bugs
+**************
+
```

Comment:
I think an introductory sentence or two here would make this a little easier for newer folks to digest. If we immediately launch into `collect-logs` stuff, people may bounce off, whereas with a little context they'll understand why we want them to collect those logs.

Reply:
Agreed I added an quick intro as to what this document will address